frontend/src/static/css/style.scss

```diff
@@ -332,4 +344,21 @@ header {
   pre {
     margin: 0;
   }
+
+  span.loc {
+    color: mui-color('grey');
+  }
+
+  .message-title {
+    display: inline;
+    font-family: monospace;
+    .within-border {
+      display: inline;
+    }
+    .not-within-border {
+      border-left: 1px dotted mui-color('black'); // 50th character line
+      display: inline;
+      pointer-events: none;
```

Comment:
Why do we need the pointer-events none here? It doesnt seem to produce any difference on my computer.

Reply:
because users might want to select and copy&paste the chunks of code, without pointer-events none, we are unable to select the code.

Reply:
Just one nit. I am okay with the style name `not-within-border`. But I think `outside-border` might be better. Can go ahead with whatever other reviewers agree too with.

Reply:
> I understand that we need this for the pre part to select the code body, but this one is for the title? I can still select the part of title I want without this property.

Oops my bad, I thought u were referring to the message body. Yes in this case, the `pointer-events:none` is unnecessary. Thanks for pointing it out!